packages/nuxeo-web-ui-ftest/features/step_definitions/bulk_edit.js

```diff
@@ -0,0 +1,44 @@
+import { Then } from '@cucumber/cucumber';
+
+Then('I click the edit button', function() {
+  this.ui.bulkEdit.editDocumentsButton.click();
+});
+
+Then('I see a dialog with editing options', function() {
+  this.ui.bulkEdit.dialog.isVisible().should.be.true;
+  this.ui.bulkEdit.bulkDefaultLayout.isVisible().should.be.true;
```

Comment:
Would recommend for stability:
```suggestion
  this.ui.bulkEdit.dialog.waitForDisplayed();
  this.ui.bulkEdit.bulkDefaultLayout.waitForDisplayed();
```